picoscope/ps4000.py

```diff
@@ -285,7 +292,7 @@ def _lowLevelGetTimebase(self, tb, noSamples, oversample, segmentIndex):
 
         return (sampleRate.value / 1.0E9, maxSamples.value)
 
```

Comment:
You should add 
```python
    def getTimeBaseNum(self, sampleTimeS):
        """Return sample time in seconds to timebase as int for API calls."""
        raise NotImplementedError('Timebase functions have not been '
                                  'written for the scope.')

    def getTimestepFromTimebase(self, timebase):
        """Return timebase to sampletime as seconds."""
        raise NotImplementedError('Timebase functions have not been '
                                  'written for the scope.')
```

So that it appears in the documentation